lib/runtime/wasmer/exports.go

```diff
@@ -110,13 +113,18 @@ func (in *Instance) BabeGenerateKeyOwnershipProof(slot uint64, authorityID [32]b
 		return nil, fmt.Errorf("executing %s: %w", runtime.BabeAPIGenerateKeyOwnershipProof, err)
 	}
 
-	keyOwnershipProof := types.OpaqueKeyOwnershipProof{}
+	var keyOwnershipProof *types.OpaqueKeyOwnershipProof
 	err = scale.Unmarshal(encodedKeyOwnershipProof, &keyOwnershipProof)
```

Comment:
```suggestion
	err = scale.Unmarshal(encodedKeyOwnershipProof, keyOwnershipProof)
```

keyOwnershipProof is already a pointer. `&keyOwnershipProof` here becomes a pointer to pointer.

Reply:
Yes that is the intended behavior now. Scale takes a pointer to the object, which in this case is a pointer, so a pointer to a pointer is intentional.